mongo_connector/oplog_manager.py

```diff
@@ -574,7 +501,8 @@ def dump_collection(self):
         dump_cancelled = [False]
 
         def get_all_ns():
-            all_ns_set = []
+            ns_set = []
```

Comment:
Changed `get_all_ns` to return separate lists for gridfs collections and non-gridfs-collections.